config.example.yml

```diff
@@ -2,11 +2,13 @@
 # Save it to `config.yml` when edited
 
 server:
+  listenaddr: "" # the address to bind on, leave empty to bind on all addresses
   port: 80 # the port the HTTP server will listen on
 
   ssl:
     enabled: false # if https should be enabled
     redirecttohttps: true # redirect to https if site is accessed by http
+    listenaddr: "" # the address to bind on, leave empty to bind on all addresses
```

Comment:
Do we need the listen addr two times for http and https or is one enough?

Reply:
I also don't see a case where we would be listening on different addresses, but the current config structure makes HTTP and HTTPS config somewhat mangled(http listen options is in .server while HTTPS listen options is in .server.ssl) so I kept both to prevent confusion.